test/Deck-test.js

```diff
@@ -19,22 +20,18 @@ describe('Deck', function() {
     });
 
     it('should store a collection of playable card objects', function() {
-        const card1 = new Card();
-        const card2 = new Card();
-        const card3 = new Card();
+        let playCards = data.prototypeData;
```

Comment:
I am glad I changed this from creating object instances into just pulling the data from the data file.